tests/rbd_mirror/test_rbd_mirror_journal_to_snap.py

```diff
@@ -0,0 +1,77 @@
+from ceph.parallel import parallel
+from tests.rbd.exceptions import RbdBaseException
+from tests.rbd_mirror import rbd_mirror_utils as rbdmirror
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(**kw):
+    """verification Snap mirroring on journaling based image after disable journaling on image.
+
+    Args:
+        **kw:
+    Returns:
+        0 - if test case pass
+        1 - it test case fails
+
+    Test case flow:
+    1. Create image on cluster-1.
+    2. Enable journal mirroring for created image on cluster-1 and run IOs.
+    3. Ensure mirror image created on cluster-2.
+    4. Disable journal mirroring on image.
+    5. enable snapshot mirroring on same image.
+    6. Verify for snapshot mirroring and run IOs
+    7. cleanup
+    """
+    try:
+        log.info("Starting RBD mirroring test case")
+        config = kw.get("config")
+        mirror1, mirror2 = [
+            rbdmirror.RbdMirror(cluster, config)
+            for cluster in kw.get("ceph_cluster_dict").values()
+        ]
+        poolname = mirror1.random_string() + "_tier_2_rbd_mirror_pool"
```

Comment:
Are we covering both EC pool and replicated pool for this journal to snapshot mirror conversion?

Reply:
right now we are not covering ec pool for rbd_mirror. As discussed in peer-review meeting.